### DISCUSSION TITLE
Navigation Controller support

### DISCUSSION THREAD

**Discussion by DMTwo** [nefarius/DsHidMini#199] [Support]
@nefarius 
I really appreciatte your job bringing the Windows support to PS3 accesorries!
Thank you very much!

I`ve saw that you have written some lines on dshidmini.inf regarding the Navigation controller, however they are commented, so I tried to force install it but unfortunatelly I could not manage to install the PS3 Navigation controller...

Is there any way to force it?
Could you give a little help?

I love to use my Navigation controller combined to my mouse, using this I have the best of two worlds.
Today I`m using the SCPToolkit and a custom XBCD driver to install and customize the controller to DirectInput, so I can use another software (Xpadder) to bind the controller buttons to keyboard buttons.

Probably using this new piece of software I will be able to use it wireless and hopefully less buggy since I have several work around software to achieve a feature that is directly changed in the settings from your new piece of software...

Once again, thanks for your time and hope to get an answer from you soon.

**Comment by nefarius** [comment]
They're commented out for a reason, those alone are not enough to make the driver work on Nav, the implementation is not finished, therefore commented out 😉

**Comment by InterClaw** [comment]
I also have a Navigation Controller. @nefarius if you need another beta tester or something for this functionality, let me know. :thumbsup:

**Reply by nefarius** [reply]
Noted 👍

**Comment by JSRRosenbaum** [comment]
@DMTwo 

See my post here about enabling Navigation Support

https://github.com/ViGEm/DsHidMini/issues/48#issuecomment-1360003578

**Reply by DMTwo** [reply]
Thanks I gonna try!
